### PR TITLE
January MLH GHW Schedule Added 

### PR DESCRIPTION
**Pull request by Anuragwagh**
The data for January Schedule of MLH GHW in IST hs added in the src/data/ScheduleData.js file & Meta tags

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
     "@testing-library/user-event": "^13.5.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-router-dom": "^6.6.1",
```
Comment (inclinedadarsh) [inline package.json:11]: why is react router dom needed?

**Comment by inclinedadarsh** [comment]
@Anuragwagh Can you please remove the changes other than in the `ScheduleData.js`?

Actually those are some really beginner friendly changes and I was thinking that they should be done by someone who's new in Open-Source.

File: package.json (hunk 1 above)
Comment (Anuragwagh) [reply package.json:11]: Actually, I was working on another project, and I accidentally imported it. I think I forgot to remove it. No worries, I will remove it.

**Comment by Anuragwagh** [comment]
> @Anuragwagh Can you please remove the changes other than in the `ScheduleData.js`?
> 
> Actually those are some really beginner friendly changes and I was thinking that they should be done by someone who's new in Open-Source.

@inclinedadarsh  Okay

**Pull request closed without merging**